app/controllers/appointment_summaries_controller.rb

```diff
@@ -0,0 +1,65 @@
+class AppointmentSummariesController < ApplicationController
+  layout 'guides'
+
+  helper_method :headers
+  helper_method :appointment_type
+
+  def new
+    @appointment_summary = AppointmentSummary.new
+  end
+
+  def create
+    appointment_summary = AppointmentSummary.new(appointment_summary_params)
+    output_document = OutputDocument.new(appointment_summary)
```

Comment:
Maybe we could rename `OutputDocument` to something that makes more sense within the domain terms of this application?

`OutputDocument` is essentially a [factory](https://practicingruby.com/articles/creational-design-patterns) – it allows us to manufacture a HTML or PDF representation of the appointment summary but without needing knowledge of how the formats are created.